### PR TITLE
Fire beacon

### PR DESCRIPTION
**Pull request by keirog**

### CODE REVIEW

File: lib/beacon.js [hunk 1]
```diff
@@ -15,10 +16,39 @@ var Beacon = function (url) {
...
+//TODO: split model bits out
+Beacon.prototype.fire = function (eventType, options) {
 	if (!eventType) {
 		throw new Error('Expected event type');
 	}
+
+	var headers = options.headers || {};
+	var erights = headers['X-FT-User-Id'] || 'unknown';
+	var isStaff = (typeof headers['X-FT-Staff'] === 'undefined') ? headers['X-FT-Staff'] : 'unknown';
+	var flags = headers['X-Requested-Flags'] || 'unknown';
```
Comment (commuterjoy) [inline lib/beacon.js]: not `x-flags` ?

Comment (keirog) [inline lib/beacon.js]: Oops, ta @commuterjoy

File: app.js [hunk 2]
```diff
@@ -3,15 +3,4 @@
...
-exports.Beacon = Beacon;
-
-function wrap(url) {
-	return create(url);
-}
-
-function create(url) {
-	return new Beacon(url);
-}
+module.exports = new Beacon('http://ft-next-bacon.herokuapp.com.global.prod.fastly.net/px.gif/');
```
Comment (commuterjoy) [inline app.js]: maybe env var?

**Comment by commuterjoy** [comment]
:+1: 

I think this is fine. I think it needs to be used *after* any response to the user, otherwise we are pegging our performance/stability to the Keen API, which would be bad. Perhaps write that in the README.

The other option is to adopt a model like next-metrics, which collects metrics over 30 seconds, then pushes them up to the API.

**Pull request merged**